internify/src/components/pages/Create.js

```diff
@@ -67,13 +68,15 @@ function Create(props) {
     props.actions.getStudents();
   }, [])
 
+
   function updateStore() {
+
     setCurrentStep(currentStep + 1);
-    console.log(jobData);
-    // Redirects view to top
+    props.actions.addJobsData(jobData);
```

Comment:
was it part of this story to make the connection to addJobsData? Shouldn't this be in a separate story? We haven't fully decided if we want to add the job to mongoDB each time we click create bc we are going to clog the db with a bunch of test jobs.

Reply:
- you are correct it was not part of the user story, and i can definitely remove it. It was added just so that feedback would be available during the process of removing persistence.  without that line of code, you do not get to see the jobPost object as it is being updated.  

- note that a warning on the console appears when this is removed ("React Hook useEffect has a missing dependency: 'props.actions'